Classes/Mvc/RequestBuilder.php

```diff
@@ -27,7 +27,11 @@ class RequestBuilder
      */
     public function build($rawRequestArgument): Request
     {
-        if (empty($rawRequestArgument['context']) || !is_string($rawRequestArgument['context'])) {
+        if (
+            ! array_key_exists('context', $rawRequestArgument)
+            || empty($rawRequestArgument['context'])
```

Comment:
the `array_key_exists` is superfluous, isn't it? `empty` does an `isset()` internally already.

Reply:
I got a PHP warning without using `array_key_exists` or `isset` before calling `empty` with PHP 8.2

Reply:
hmm that's weird…
I'm pretty sure that I got an error at exactly that point last week, but now I can't reproduce it any more…

let's pretend this never happened 😅